console/console-init/ui/src/graphql-module/queries/iot_device.ts

```diff
@@ -44,4 +46,87 @@ const DELETE_IOT_DEVICE = gql`
   }
 `;
 
-export { RETURN_IOT_DEVICE_DETAIL, RETURN_IOT_CREDENTIALS, DELETE_IOT_DEVICE };
+const RETURN_ALL_DEVICES_FOR_IOT_PROJECT_SORT = (sortBy?: ISortBy) => {
+  let orderBy = "";
+  if (sortBy) {
+    switch (sortBy.index) {
+      case 0:
```

Comment:
I think we don't need this case if not doing anything. This kind of cases hanlded by default case.